DistributionPackages/Sfi.Kateheo/Resources/Private/Templates/Authentication/index.html

```diff
@@ -1,5 +1,5 @@
 {namespace neos=Neos\Neos\ViewHelpers}
-<div class="login-page">
+<div class="login-page" style="margin: 48px 12px">
```

Comment:
```suggestion
<div style="margin: 48px 12px">
```
Let's just remove it, since it's not used anywhere now.